30-remote-site-setup.sh

```diff
@@ -41,20 +43,18 @@ foreach_bosco_endpoint setup_ssh_config
 ENDPOINT_CONFIG=/etc/endpoints.ini
 setup_endpoints_ini () {
     remote_home_dir=$(ssh -i $BOSCO_KEY ${ruser}@${rhost} pwd)
+    remote_os_ver=$(ssh -i $BOSCO_KEY ${ruser}@${rhost} "rpm -E %rhel")
     cat <<EOF >> $ENDPOINT_CONFIG
 [Endpoint ${RESOURCE_NAME}-${ruser}]
 local_user = ${ruser}
 remote_host = ${rhost}
 remote_user = ${ruser}
 remote_dir = $remote_home_dir/bosco-osg-wn-client
+upstream_url = https://repo.opensciencegrid.org/tarball-install/3.4/osg-wn-client-latest.${remote_os_ver}.x86_64.tar.gz
```

Comment:
```suggestion
upstream_url = https://repo.opensciencegrid.org/tarball-install/3.4/osg-wn-client-latest.el${remote_os_ver}.x86_64.tar.gz
```

So, `rpm -E %rhel` just returns the int, eg `6` or `7`, but these tarballs are named (eg) `osg-wn-client-latest.el6.x86_64.tar.gz`